src/test/compile-fail/borrowck/two-phase-activation-sharing-interference.rs

```diff
@@ -10,10 +10,12 @@
 
 // ignore-tidy-linelength
 
-// revisions: lxl_beyond nll_beyond nll_target
+// revisions: nll_target
 
+// The following revisions are disabled due to missing support from two-phase beyond autorefs
```

Comment:
I actually don't think you should worry about trying to support 2-phase borrow beyond autoref.

I had originally thought the general idea could make sense, but at this point I think there are enough cases where it does not work that it would not be a good use of time to try to hack it together. Or at least, I'd want to see a more theoretically proof of soundness for the idea before we implemented anything.